scripts/common/results/result_info.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Structures for a variety of different test results."""
+
+
+def build_test_result(test_id,
+                      result,
+                      result_type='total_time',
```

Comment:
I would not make this a default value since it will force users to think though what they are storing. Its possible users will store time but this may be total time per epoch, total time for the entire training run etc. It might be good for users to explicitly specify this.